cannot build from Dockerfile

When building from the Dockerfile:

```
Step 17/33 : RUN source /opt/qt512/bin/qt512-env.sh && qmake ../src && make
 ---> Running in 1e7bea85745c
Info: creating stash file /dsistudio/build/.qmake.stash
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
/opt/qt512/bin/uic ../src/mainwindow.ui -o ui_mainwindow.h
/opt/qt512/bin/uic: error while loading shared libraries: libQt5Core.so.5: cannot open shared object file: No such file or directory
make: *** [ui_mainwindow.h] Error 127
Makefile:6207: recipe for target 'ui_mainwindow.h' failed
The command '/bin/bash -c source /opt/qt512/bin/qt512-env.sh && qmake ../src && make' returned a non-zero code: 2
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -36,6 +36,7 @@ RUN mv frankyeh-TIPL-* src/tipl
 
 RUN mkdir build
 WORKDIR build
+RUN strip --remove-section=.note.ABI-tag /opt/qt512/lib/libQt5Core.so.5
 RUN source /opt/qt512/bin/qt512-env.sh && qmake ../src && make
 
 WORKDIR ..

